packages/react-core/src/components/Menu/examples/MenuOptionSingleSelect.tsx

```diff
@@ -12,9 +12,9 @@ export const MenuOptionSingleSelect: React.FunctionComponent = () => {
   };
 
   return (
-    <Menu onSelect={onSelect} activeItemId={activeItem} selected={selectedItem}>
+    <Menu role="listbox" onSelect={onSelect} activeItemId={activeItem} selected={selectedItem}>
```

Comment:
Can we note the use of `listbox` in the text above these two updated examples? since erin has written such beautiful documentation for our Menu examples 😊